tests/nataliia_romanchuk/homework_5/test_set.py

```diff
@@ -0,0 +1,80 @@
+# SETS
+
+some_set = {"a", "b", "c"}
+some_set_2 = {"b", "a", "c"}
+
+
+def test_is_container():
+    assert len(some_set) == 3
+    assert "a" in some_set
+    assert list(sorted(some_set)) == ['a', 'b', 'c']
+
+
+def test_is_subset1():
+    assert some_set == some_set_2
+    assert sorted(some_set) <= sorted(some_set_2)
+    assert sorted(some_set_2) <= sorted(some_set)
+
+    assert {"a"} < some_set
+    # assert {"f"} < some_set           ##don't know how to test it
```

Comment:
<     is for subset

that means all elements from 1 set are contained in another set

assert not {"f"} < some_set 
assert not {"f", "a"} < some_set 
assert not {"a", "b", "c"} < some_set 


assert {"a"} < some_set 
assert {"a", "b"} < some_set 
assert not {"a", "b", "c"} <= some_set 
assert not {"a", "b", "c"} == some_set

Reply:
added